internal/Get-Connection.ps1

```diff
@@ -0,0 +1,115 @@
+﻿Function Get-Connection
```

Comment:
sorry, I mixed some functions up:
It's similar to Connect-SqlServer
So, would Connect-Computer be more consisent?

Test-Sqlconnection is with the more extensive tests and info, which we're also working on.